gulpfile.js

```diff
@@ -33,6 +35,20 @@ gulp.task('clean', (done) => {
 
 gulp.task('build-all', ['clean', 'init', 'build']);
 
+gulp.task('lint', function (cb) {
+  exec('depcheck . --ignores "generator-polymer-init"', function (err, stdout, stderr) {
```

Comment:
re global install: whoops, I completely missed that implication. I think this is a step backwards then (npm was including the dev dependency in the path when run). i'll close this until it's ready to go again.